### DISCUSSION TITLE
'Is a' Relationship in Conceptual Model

### DISCUSSION THREAD

**Discussion by mehdihadeli** [kgrzybek/modular-monolith-with-ddd#160] [Q&A]
Hi,
I have a question about your Conceptual model,
Why in this model `Member` has a `Is a` relationship to `User`?
Our `Member` class not inherits from user directly and my question is why you use this 'is a' relationship here

Thanks

**Comment by AndreiGanichev** [comment]
Hi, @mehdihadeli 
Although the conceptual model is made as UML class diagram it's also includes existed Bounded Contexts(BC). It's essential for DDD to have for the same or close concepts(one who use the application in current example) **different** models  in different BCs . And these models must precisely reflect the Ubiquitous Language(UL) using into and bounded by a BC. So"is a" relation between Member and User is a semantic association. 
UL of the Meeting context doesn't contain a term _user_. But in the same time we can't create a Member in the Meeting context out of nowhere but we have to have a registered User in the User Access context to create a Member from.
